tests/serverless_azure/sample_application/function_app.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import newrelic.agent
+
+newrelic.agent.initialize()  # Initialize the New Relic agent
```

Comment:
Is this manual agent initialization required? The `collector_agent_registration_fixture` does an `initialize` call also.

Reply:
This is fixed